FlyingFox/Sources/HTTPRoute.swift

```diff
@@ -167,12 +223,39 @@ public extension HTTPRoute {
         }
     }
 
-    private static func components(for target: String) -> (method: String, path: String) {
+    private static func components(for target: String) -> (
+        methods: Set<HTTPMethod>,
+        path: String
+    ) {
         let comps = target.split(separator: " ", maxSplits: 1, omittingEmptySubsequences: true)
```

Comment:
Now that we accept a comma delimited list of methods:
`GET,PUT /fish` -> `(method: [.GET, .PUT], path: "/fish")`

 if a user adds a single whitespace to their list it splits into the path:
`GET, PUT /fish` -> `(method: [.GET], path: "PUT /fish")`

I asked ChatGPT to write a regex for the split and it confidently spat out a regex that didn't do what I wanted 😩. 
If it proves an issue, I might be able to circle back with a NSScanner based split as I always find that more understandable.



        let comps = target.split(separator: " ", maxSplits: 1, omittingEmptySubsequences: true)